metabulo/views.py

```diff
@@ -529,3 +530,39 @@ def get_pca_overview(validated_table):
     png_content = opencpu.generate_image('/metabulo/R/pca_overview_plot',
                                          validated_table.measurements)
     return Response(png_content, mimetype='image/png')
+
+
+@csv_bp.route('/csv/<uuid:csv_id>/analyses/wilcoxon', methods=['GET'])
+@use_kwargs({
+    'zero_method': fields.Str(missing='wilcox',
+                              validate=validate.OneOf(['wilcox', 'pratt', 'zsplit'])),
+    'alternative': fields.Str(missing='two-sided',
```

Comment:
Is `two-sided` a specific enough name? Or is that a qualifier for different kinds of tests (e.g., "two-sided t-test")?

Reply:
Offloaded to https://github.com/girder/viime/issues/228